gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -1226,3 +1226,6 @@ L.disable_spectatorsoutline_tip = "Deaktiviert die Umrandungen um ein Objekt, in
 
 L.disable_overheadicons = "Deaktiviere Rollenicons"
 L.disable_overheadicons_tip = "Deaktiviert die Rollenicons über den Köpfen der Spieler (+Performance)"
+
+L.disable_doubletap_sprint = "Deaktiviere 'doubletap' Sprinten"
+L.disable_doubletap_sprint_tip = "Deaktiviert das Sprinten wenn die Nach-Vorne-Gehen Taste doppelt gedrückt wird"
```

Comment:
I would go with "Deaktiviere Double-Tap Sprinten" instead. Maybe "Vorwärtstaste" would be better. If my [PR](https://github.com/TTT-2/TTT2/pull/435) is merged, this should be changed to "Deaktiviert das Auslösen des Sprinten durch Double-Tap einer Bewegungstaste", since it could be any movement key.